src/bc.F

```diff
@@ -3409,209 +3409,193 @@ subroutine bcs_GPU(s)
       real, dimension(ib:ie,jb:je,kb:ke) :: s
 
       integer i,j,k
+      !$acc declare present(s)
 
-      !$acc data present(s)
```

Comment:
All my changes to `bc.F` are reverted. I would suggest to checking out my version and replacing your existing one.